media-fonts/b612-font/b612-font-1.003_p20171129.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit font vcs-snapshot
+
+DESCRIPTION="Font designed for aircraft cockpit displays"
+HOMEPAGE="http://b612-font.com/"
+SRC_URI="https://git.polarsys.org/c/b612/b612.git/snapshot/b612-bd14fde2544566e620eab106eb8d6f2b7fb1347e.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
I don't think you can add arm and arm64 keywords to new packages.

Reply:
Too bad

Reply:
you can, if you can test on arm/arm64 yourself.